client/app/components/sections/Topbar/Topbar.js

```diff
@@ -10,11 +10,11 @@ class Topbar extends Component {
   render() {
     return div({ className: css.topbar }, [
       r(Logo, { ...this.props.logo, classSet: css.topbarLogo }),
-      this.props.search_mode ?
+      this.props.search ?
         r(SearchBar, {
-          mode: this.props.search_mode,
-          keywordPlaceholder: this.props.search_keyword_placeholder,
-          locationPlaceholder: this.props.search_location_placeholder,
+          mode: this.props.search.mode,
+          keywordPlaceholder: this.props.search.keyword_placeholder,
+          locationPlaceholder: this.props.search.location_placeholder,
```

Comment:
I like the fact that we use camelCase in presentational components, but I want to leave further adjustments for Kimmo to be done on his own branch.